diff --git a/dist/index.browser.cjs b/dist/index.browser.cjs
index 69667b3726bdc002a3ad2ce6f6bd6b2943c60896..fb9cd4c27f6d808c13e762c387c3360461dae650 100644
--- a/dist/index.browser.cjs
+++ b/dist/index.browser.cjs
@@ -101,7 +101,7 @@ function getRpcTransportWithRequestCoalescing(transport, getDeduplicationKey) {
     if (signal) {
       const responsePromise = coalescedRequest.responsePromise;
       return await new Promise((resolve, reject) => {
-        const handleAbort = (e) => {
+        const handleAbort = () => {
           signal.removeEventListener("abort", handleAbort);
           coalescedRequest.numConsumers -= 1;
           queueMicrotask(() => {
@@ -110,7 +110,7 @@ function getRpcTransportWithRequestCoalescing(transport, getDeduplicationKey) {
               abortController.abort(EXPLICIT_ABORT_TOKEN ||= createExplicitAbortToken());
             }
           });
-          reject(e.target.reason);
+          reject(signal.reason);
         };
         signal.addEventListener("abort", handleAbort);
         responsePromise.then(resolve).catch(reject).finally(() => {
diff --git a/dist/index.browser.mjs b/dist/index.browser.mjs
index d9cc20fbdd9a05fcc029a78e340ebc51470006a0..ae2fdb33ea1788da22c9a477d547159f53c4ffcb 100644
--- a/dist/index.browser.mjs
+++ b/dist/index.browser.mjs
@@ -97,7 +97,7 @@ function getRpcTransportWithRequestCoalescing(transport, getDeduplicationKey) {
     if (signal) {
       const responsePromise = coalescedRequest.responsePromise;
       return await new Promise((resolve, reject) => {
-        const handleAbort = (e) => {
+        const handleAbort = () => {
           signal.removeEventListener("abort", handleAbort);
           coalescedRequest.numConsumers -= 1;
           queueMicrotask(() => {
@@ -106,7 +106,7 @@ function getRpcTransportWithRequestCoalescing(transport, getDeduplicationKey) {
               abortController.abort(EXPLICIT_ABORT_TOKEN ||= createExplicitAbortToken());
             }
           });
-          reject(e.target.reason);
+          reject(signal.reason);
         };
         signal.addEventListener("abort", handleAbort);
         responsePromise.then(resolve).catch(reject).finally(() => {
diff --git a/dist/index.native.mjs b/dist/index.native.mjs
index 8ced7dfe02a51ee14f946f9d132a8f82bf66505e..75ec23a9e78c118b255343251934612ef3940542 100644
--- a/dist/index.native.mjs
+++ b/dist/index.native.mjs
@@ -97,7 +97,7 @@ function getRpcTransportWithRequestCoalescing(transport, getDeduplicationKey) {
     if (signal) {
       const responsePromise = coalescedRequest.responsePromise;
       return await new Promise((resolve, reject) => {
-        const handleAbort = (e) => {
+        const handleAbort = () => {
           signal.removeEventListener("abort", handleAbort);
           coalescedRequest.numConsumers -= 1;
           queueMicrotask(() => {
@@ -106,7 +106,7 @@ function getRpcTransportWithRequestCoalescing(transport, getDeduplicationKey) {
               abortController.abort(EXPLICIT_ABORT_TOKEN ||= createExplicitAbortToken());
             }
           });
-          reject(e.target.reason);
+          reject(signal.reason);
         };
         signal.addEventListener("abort", handleAbort);
         responsePromise.then(resolve).catch(reject).finally(() => {
diff --git a/dist/index.node.cjs b/dist/index.node.cjs
index f312cf602bbecc3603cb5227ac7cfc49e9653607..aba0d7573f73ced05c70f1dce3c6e0213c8865c2 100644
--- a/dist/index.node.cjs
+++ b/dist/index.node.cjs
@@ -104,7 +104,7 @@ function getRpcTransportWithRequestCoalescing(transport, getDeduplicationKey) {
     if (signal) {
       const responsePromise = coalescedRequest.responsePromise;
       return await new Promise((resolve, reject) => {
-        const handleAbort = (e2) => {
+        const handleAbort = () => {
           signal.removeEventListener("abort", handleAbort);
           coalescedRequest.numConsumers -= 1;
           queueMicrotask(() => {
@@ -113,7 +113,7 @@ function getRpcTransportWithRequestCoalescing(transport, getDeduplicationKey) {
               abortController.abort(EXPLICIT_ABORT_TOKEN ||= createExplicitAbortToken());
             }
           });
-          reject(e2.target.reason);
+          reject(signal.reason);
         };
         signal.addEventListener("abort", handleAbort);
         responsePromise.then(resolve).catch(reject).finally(() => {
diff --git a/dist/index.node.mjs b/dist/index.node.mjs
index c94bde7a06fd9e44894faf331b7e8649766240b6..6c2f76d3ecefe9044eda4b0b3d3d2e2d49ea0ba6 100644
--- a/dist/index.node.mjs
+++ b/dist/index.node.mjs
@@ -100,7 +100,7 @@ function getRpcTransportWithRequestCoalescing(transport, getDeduplicationKey) {
     if (signal) {
       const responsePromise = coalescedRequest.responsePromise;
       return await new Promise((resolve, reject) => {
-        const handleAbort = (e2) => {
+        const handleAbort = () => {
           signal.removeEventListener("abort", handleAbort);
           coalescedRequest.numConsumers -= 1;
           queueMicrotask(() => {
@@ -109,7 +109,7 @@ function getRpcTransportWithRequestCoalescing(transport, getDeduplicationKey) {
               abortController.abort(EXPLICIT_ABORT_TOKEN ||= createExplicitAbortToken());
             }
           });
-          reject(e2.target.reason);
+          reject(signal.reason);
         };
         signal.addEventListener("abort", handleAbort);
         responsePromise.then(resolve).catch(reject).finally(() => {
diff --git a/src/rpc-request-coalescer.ts b/src/rpc-request-coalescer.ts
index 9d83032ea47141be79e3cb3d424cb4657a2904c6..67e7a481ba41534d4782de72b48c69f3bc2a47f0 100644
--- a/src/rpc-request-coalescer.ts
+++ b/src/rpc-request-coalescer.ts
@@ -75,7 +75,7 @@ export function getRpcTransportWithRequestCoalescing<TTransport extends RpcTrans
         if (signal) {
             const responsePromise = coalescedRequest.responsePromise as Promise<RpcResponse<TResponse>>;
             return await new Promise<RpcResponse<TResponse>>((resolve, reject) => {
-                const handleAbort = (e: AbortSignalEventMap['abort']) => {
+                const handleAbort = () => {
                     signal.removeEventListener('abort', handleAbort);
                     coalescedRequest.numConsumers -= 1;
                     queueMicrotask(() => {
@@ -85,7 +85,7 @@ export function getRpcTransportWithRequestCoalescing<TTransport extends RpcTrans
                         }
                     });
                     // eslint-disable-next-line @typescript-eslint/prefer-promise-reject-errors
-                    reject((e.target as AbortSignal).reason);
+                    reject(signal.reason);
                 };
                 signal.addEventListener('abort', handleAbort);
                 responsePromise
