diff --git a/android/src/main/java/expo/modules/updates/UpdatesPackage.kt b/android/src/main/java/expo/modules/updates/UpdatesPackage.kt
index c1b0c44fe990087cf7b40eb80e8ceed18cbf1a30..68a73a0ddb98104ff055084415822052ef017e64 100644
--- a/android/src/main/java/expo/modules/updates/UpdatesPackage.kt
+++ b/android/src/main/java/expo/modules/updates/UpdatesPackage.kt
@@ -103,7 +103,10 @@ class UpdatesPackage : Package {
     val handler = object : ApplicationLifecycleListener {
       override fun onCreate(application: Application) {
         super.onCreate(application)
-        if (isRunningAndroidTest()) {
+       /**
+        * This fix is needed for running Detox tests with Android debug build.
+        */
+        if (isRunningAndroidTest() && !BuildConfig.USE_DEV_CLIENT) {
           // Preload updates to prevent Detox ANR
           UpdatesController.initialize(context)
           UpdatesController.instance.launchAssetFile
