diff --git a/node_modules/@types/invity-api/index.d.ts b/node_modules/@types/invity-api/index.d.ts
index 0ca6a40..f8d2711 100755
--- a/node_modules/@types/invity-api/index.d.ts
+++ b/node_modules/@types/invity-api/index.d.ts
@@ -239,6 +239,7 @@ export interface ExchangeTrade {
     extraFieldDescription?: CoinExtraField | undefined;
     tags?: ExchangeTradeTag[] | undefined;
     id?: string | undefined; // internal DB id
+    infoNote?: string | undefined;
 
     // DEX extra fields
     isDex?: boolean | undefined;
