diff --git a/node_modules/app-builder-lib/out/macPackager.js b/node_modules/app-builder-lib/out/macPackager.js
index a52a853..216a77d 100644
--- a/node_modules/app-builder-lib/out/macPackager.js
+++ b/node_modules/app-builder-lib/out/macPackager.js
@@ -474,6 +474,9 @@ class MacPackager extends platformPackager_1.PlatformPackager {
         const extendInfo = this.platformSpecificBuildOptions.extendInfo;
         if (extendInfo != null) {
             Object.assign(appPlist, extendInfo);
+            for(const [k,v] of Object.entries(appPlist)) {
+                if(v === null || v === undefined) delete appPlist[k]
+            }
         }
     }
     async signApp(packContext, isAsar) {
