diff --git a/node_modules/app-builder-lib/out/targets/ArchiveTarget.js b/node_modules/app-builder-lib/out/targets/ArchiveTarget.js
index 72a59df..9095aad 100644
--- a/node_modules/app-builder-lib/out/targets/ArchiveTarget.js
+++ b/node_modules/app-builder-lib/out/targets/ArchiveTarget.js
@@ -120,7 +120,7 @@ class ArchiveTarget extends _core().Target {
       await (0, _archive().archive)(format, artifactPath, dirToArchive, archiveOptions);
 
       if (this.isWriteUpdateInfo && format === "zip") {
-        updateInfo = await (0, _differentialUpdateInfoBuilder().appendBlockmap)(artifactPath);
+        // updateInfo = await (0, _differentialUpdateInfoBuilder().appendBlockmap)(artifactPath);
       }
     }
 
