diff --git a/node_modules/nextra/dist/loader.mjs b/node_modules/nextra/dist/loader.mjs
index a14e959..98b314a 100644
--- a/node_modules/nextra/dist/loader.mjs
+++ b/node_modules/nextra/dist/loader.mjs
@@ -28,6 +28,10 @@ var initGitRepo = (async () => {
     const { Repository } = await import("@napi-rs/simple-git");
     try {
       const repository = Repository.discover(CWD);
+      if(repository.isWorktree()) {
+        // doesn't work inside git worktree
+        return {};
+      }
       if (repository.isShallow()) {
         if (process.env.VERCEL) {
           console.warn(
