diff --git a/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java b/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java
index 15afbed..5a4f50e 100644
--- a/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java
+++ b/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java
@@ -102,9 +102,11 @@ public class AnimationsManager implements ViewHierarchyObserver {
       return;
     }
 
-    AndroidUIScheduler androidUIScheduler = mWeakAndroidUIScheduler.get();
-    if (androidUIScheduler != null) {
-      androidUIScheduler.triggerUI();
+    if (mWeakAndroidUIScheduler != null) {
+      AndroidUIScheduler androidUIScheduler = mWeakAndroidUIScheduler.get();
+      if (androidUIScheduler != null) {
+        androidUIScheduler.triggerUI();
+      }
     }
     int tag = view.getId();
     HashMap<String, Object> targetValues = after.toTargetMap();
