diff --git a/node_modules/ripple-lib/dist/npm/api.d.ts b/node_modules/ripple-lib/dist/npm/api.d.ts
index fd47470..8c9b929 100644
--- a/node_modules/ripple-lib/dist/npm/api.d.ts
+++ b/node_modules/ripple-lib/dist/npm/api.d.ts
@@ -52,11 +52,13 @@ export interface APIOptions extends ConnectionUserOptions {
     maxFeeXRP?: string;
     proxy?: string;
     timeout?: number;
+    agent?: any;
 }
 declare class RippleAPI extends EventEmitter {
     _feeCushion: number;
     _maxFeeXRP: string;
     connection: Connection;
+    static _ALLOW_AGENT: boolean;
     static _PRIVATE: {
         validate: typeof validate;
         RangeSet: typeof RangeSet;
diff --git a/node_modules/ripple-lib/dist/npm/api.js b/node_modules/ripple-lib/dist/npm/api.js
index 5d48807..4ac1631 100644
--- a/node_modules/ripple-lib/dist/npm/api.js
+++ b/node_modules/ripple-lib/dist/npm/api.js
@@ -234,6 +234,7 @@ class RippleAPI extends events_1.EventEmitter {
     }
 }
 exports.RippleAPI = RippleAPI;
+RippleAPI._ALLOW_AGENT = true;
 RippleAPI._PRIVATE = {
     validate: common_1.validate,
     RangeSet: rangeset_1.default,
diff --git a/node_modules/ripple-lib/dist/npm/common/connection.js b/node_modules/ripple-lib/dist/npm/common/connection.js
index 8d46ea0..03ae2c1 100644
--- a/node_modules/ripple-lib/dist/npm/common/connection.js
+++ b/node_modules/ripple-lib/dist/npm/common/connection.js
@@ -42,6 +42,9 @@ const backoff_1 = require("./backoff");
 const INTENTIONAL_DISCONNECT_CODE = 4000;
 function createWebSocket(url, config) {
     const options = {};
+    if (config.agent != null) {
+        options.agent = config.agent;
+    }
     if (config.proxy != null) {
         const parsedURL = url_1.parse(url);
         const parsedProxyURL = url_1.parse(config.proxy);
diff --git a/node_modules/ripple-lib/dist/npm/common/schemas/input/api-options.json b/node_modules/ripple-lib/dist/npm/common/schemas/input/api-options.json
index 1da93be..6785a4b 100644
--- a/node_modules/ripple-lib/dist/npm/common/schemas/input/api-options.json
+++ b/node_modules/ripple-lib/dist/npm/common/schemas/input/api-options.json
@@ -26,6 +26,10 @@
       "format": "uri",
       "description": "URI for HTTP/HTTPS proxy to use to connect to the rippled server."
     },
+    "agent": {
+      "format": "object",
+      "description": "Proxy agent to use to connect to the rippled server."
+    },
     "timeout": {
       "type": "integer",
       "description": "Request timeout in milliseconds before considering a request to have failed. See also: connectionTimeout.",
