diff --git a/node_modules/trezor-connect/lib/device/DeviceList.js b/node_modules/trezor-connect/lib/device/DeviceList.js
index fd96f51..7e12d3c 100644
--- a/node_modules/trezor-connect/lib/device/DeviceList.js
+++ b/node_modules/trezor-connect/lib/device/DeviceList.js
@@ -84,7 +84,7 @@ function (_EventEmitter) {
 
       if (env === 'react-native') {
         if (!process.env.RN_EMULATOR) {
-          bridgeUrl = 'http://10.36.2.3:21325'; // TODO: remove this. SL laptop just for debugging
+          bridgeUrl = 'http://192.168.255.18:21325'; // TODO: remove this. SL laptop just for debugging
         } else if (process.env.RN_OS === 'android') {
           bridgeUrl = 'http://10.0.2.2:21325'; // Android emulator localhost
         }
diff --git a/node_modules/trezor-connect/lib/utils/browser.js b/node_modules/trezor-connect/lib/utils/browser.js
index 624516e..a658145 100644
--- a/node_modules/trezor-connect/lib/utils/browser.js
+++ b/node_modules/trezor-connect/lib/utils/browser.js
@@ -59,7 +59,7 @@ exports.getBrowserState = getBrowserState;
 var parseBridgeJSON = function parseBridgeJSON(json) {
   var preferred = '';
 
-  if (typeof window !== 'undefined') {
+  if (window && window.navigator && window.navigator.userAgent) {
     // Find preferred platform using bowser and userAgent
     var agent = window.navigator.userAgent;
 
