diff --git a/node_modules/trezor-connect/lib/types/trezor/management.js.flow b/node_modules/trezor-connect/lib/types/trezor/management.js.flow
index d56a98b..93f0d24 100644
--- a/node_modules/trezor-connect/lib/types/trezor/management.js.flow
+++ b/node_modules/trezor-connect/lib/types/trezor/management.js.flow
@@ -16,6 +16,7 @@ export type ApplySettings = {
     display_rotation?: 0 | 90 | 180 | 270;
     use_passphrase?: boolean;
     label?: string;
+    auto_lock_delay_ms?: number;
 }
 
 export type ApplyFlags = {
diff --git a/node_modules/trezor-connect/lib/typescript/trezor/management.d.ts b/node_modules/trezor-connect/lib/typescript/trezor/management.d.ts
index 4d2346a..44b4eba 100644
--- a/node_modules/trezor-connect/lib/typescript/trezor/management.d.ts
+++ b/node_modules/trezor-connect/lib/typescript/trezor/management.d.ts
@@ -14,6 +14,7 @@ export interface ApplySettings {
     display_rotation?: 0 | 90 | 180 | 270;
     use_passphrase?: boolean;
     label?: string;
+    auto_lock_delay_ms?: number;
 }
 
 export interface ApplyFlags {
