diff --git a/node_modules/trezor-connect/lib/typescript/backend/blockchain.d.ts b/node_modules/trezor-connect/lib/typescript/backend/blockchain.d.ts
index 72b02a0..5b3f582 100644
--- a/node_modules/trezor-connect/lib/typescript/backend/blockchain.d.ts
+++ b/node_modules/trezor-connect/lib/typescript/backend/blockchain.d.ts
@@ -143,6 +143,7 @@ export interface BlockchainAccountBalanceHistory {
     txs: number;
     received: string;
     sent: string;
+    sentToSelf?: string;
     rates: BlockchainFiatRates;
 }
 
