diff --git a/node_modules/trezor-connect/lib/typescript/trezor/protobuf.d.ts b/node_modules/trezor-connect/lib/typescript/trezor/protobuf.d.ts
index 0e5741f..0869748 100644
--- a/node_modules/trezor-connect/lib/typescript/trezor/protobuf.d.ts
+++ b/node_modules/trezor-connect/lib/typescript/trezor/protobuf.d.ts
@@ -336,7 +336,7 @@ export type TxOutputType = {
 } | {
     address?: typeof undefined;
     address_n: number[];
-    script_type: OutputScriptType;
+    script_type: Exclude<OutputScriptType, 'PAYTOOPRETURN'>;
     amount: string;
     multisig?: MultisigRedeemScriptType;
     orig_hash?: string;
@@ -1329,9 +1329,9 @@ export type Features = {
     minor_version: number;
     patch_version: number;
     bootloader_mode?: boolean | null;
-    device_id?: string | null;
-    pin_protection?: boolean;
-    passphrase_protection?: boolean;
+    device_id: string | null;
+    pin_protection: boolean;
+    passphrase_protection: boolean;
     language?: string;
     label?: string | null;
     initialized?: boolean;
@@ -1348,7 +1348,7 @@ export type Features = {
     fw_patch?: number | null;
     fw_vendor?: string | null;
     fw_vendor_keys?: string;
-    unfinished_backup?: boolean;
+    unfinished_backup: boolean;
     no_backup?: boolean;
     recovery_mode?: boolean;
     capabilities: Capability[];
